lib/ReactCrop.js

```diff
@@ -521,14 +524,14 @@ class ReactCrop extends Component {
 				<div className='ReactCrop--drag-bar ord-s' data-ord='s'></div>
 				<div className='ReactCrop--drag-bar ord-w' data-ord='w'></div>
 
-				<div className='ReactCrop--drag-handle ord-nw' data-ord='nw'></div>
-				<div className='ReactCrop--drag-handle ord-n' data-ord='n'></div>
-				<div className='ReactCrop--drag-handle ord-ne' data-ord='ne'></div>
-				<div className='ReactCrop--drag-handle ord-e' data-ord='e'></div>
-				<div className='ReactCrop--drag-handle ord-se' data-ord='se'></div>
-				<div className='ReactCrop--drag-handle ord-s' data-ord='s'></div>
-				<div className='ReactCrop--drag-handle ord-sw' data-ord='sw'></div>
-				<div className='ReactCrop--drag-handle ord-w' data-ord='w'></div>
+				{ellipse ? null : <div className='ReactCrop--drag-handle ord-nw' data-ord='nw' />}
```

Comment:
Yeah I realised that too so no worries!

On Wednesday, 3 August 2016, Arend van Beelen jr. <notifications@github.com>
wrote:

> In lib/ReactCrop.js
> <https://github.com/DominicTobias/react-image-crop/pull/50#discussion_r73321714>
> :
>
> > @@ -521,14 +524,14 @@ class ReactCrop extends Component {
> >  				<div className='ReactCrop--drag-bar ord-s' data-ord='s'></div>
> >  				<div className='ReactCrop--drag-bar ord-w' data-ord='w'></div>
> >
> > -				<div className='ReactCrop--drag-handle ord-nw' data-ord='nw'></div>
> > -				<div className='ReactCrop--drag-handle ord-n' data-ord='n'></div>
> > -				<div className='ReactCrop--drag-handle ord-ne' data-ord='ne'></div>
> > -				<div className='ReactCrop--drag-handle ord-e' data-ord='e'></div>
> > -				<div className='ReactCrop--drag-handle ord-se' data-ord='se'></div>
> > -				<div className='ReactCrop--drag-handle ord-s' data-ord='s'></div>
> > -				<div className='ReactCrop--drag-handle ord-sw' data-ord='sw'></div>
> > -				<div className='ReactCrop--drag-handle ord-w' data-ord='w'></div>
> > +				{ellipse ? null : <div className='ReactCrop--drag-handle ord-nw' data-ord='nw' />}
>
> I could, but if I were to do that consistently, the next line would look
> like this:
>
> {(!aspect || ellipse) && <div className='ReactCrop--drag-handle ord-n' data-ord='n' />}
>
> Not sure if that's really an improvement...
>
> But I see you already merged (thanks!), so I guess it's okay to keep it
> like this?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/DominicTobias/react-image-crop/pull/50/files/d2066a20d1d550f71f8934317a0d77f799210ccd#r73321714>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAuZ-hRfiCT33b43pkfdG7-Ow8GZmpLxks5qcHq-gaJpZM4JSrOB>
> .
>